paasta_tools/cli/cmds/info.py

```diff
@@ -37,7 +37,7 @@
 )
 
 # modes that depend on smartstack port cannot be tested via paasta proxies, so we exclude those
-TESTABLE_SERVICE_MODES = {"http", "https"}
+TESTABLE_SERVICE_MODES = {"http", "https", "http2"}
```

Comment:
`get_smartstack_endpoints()` will probably need some special handling since that passes through the mode as the uri scheme - but the http2 scheme is still `http`

Reply:
and i think the same needs to happen in `get_deployments_strings()`?